src/lib/components/overlay/table-overlay-body.vue

```diff
@@ -0,0 +1,143 @@
+<template>
```

Comment:
@sannidhi09 Overlay means to show a kind of cover while the data is loading. Just to show some blocks while the data is fetching. So, what are you trying to do here? I guess you should move this to blocks.

Reply:
We are using the `overlay-table`, so the child of this will inherit property. That's why I have created under same root.

Reply:
Move this to blocks Krishna. You could use the overlays inside that if needed.

Reply:
Okay

Reply:
Okay.